graph-db/src/utils/indexing/indexCmrCollection.js

```diff
@@ -0,0 +1,38 @@
+const { indexRelatedUrl } = require('./indexRelatedUrl');
+
+exports.indexCmrCollection = async (collection, gremlin) => {
+  const { meta, umm } = collection;
+  const conceptId = meta['concept-id'];
```

Comment:
Can destructure variables like this using this sytnax:

`const { 'concept-id': conceptId } = meta`